cmd/definitions/tmpl/service.tmpl

```diff
@@ -98,6 +98,14 @@ func setStorageSystemMetadata(s *StorageMeta, sm StorageSystemMetadata) {
 // With{{ $pname }} will apply {{ $v.Name }} value to Options.
 //
 {{ $v.Description }}
+{{- if hasPrefix $v.Name "enable_" }}
```

Comment:
Maybe we can check the `bool` type instead?

Is it OK to generate `EnableXxx()` or `DisableXxxx()` for all services?

Reply:
There's `disable_uri_cleaning` pair in our test toml. `EnableDisableURICleaning` or `DisableDisableURICleaning` looks strange. Take this scenario into consideration, how about consider only features for now and modify the condition without using prefix？

Reply:
Well, here is some misunderstanding.

- features will always be generated into `EnableXxxx` as described in GSP.
- Difference here: `disable_uri_cleaning` will be generated into `DisableURICleaning()` instead of `DisableURICleaning(value bool)`

`disable_uri_cleaning` here is a pair instead feature, so we will not add the prefix `enable_` for it.

Reply:
Got it.